src/test/data/JsonSerializableAddressBookTest/invalidFishAddressBook.json

```diff
@@ -2,7 +2,7 @@
   "fishes": [ {
     "name": "Hans Muster",
     "lastFedDate": "in/valid/date",
-    "email": "invalid@email!3e",
+    "species": "invalid314species",
```

Comment:
This is a valid species name as the validation regex of `Species` accepts alphanumeric characters.